bootstore/src/schemes/v0/peer.rs

```diff
@@ -62,6 +62,15 @@ pub enum NodeRequestError {
     },
 }
 
+impl From<NodeRequestError> for omicron_common::api::external::Error {
+    fn from(error: NodeRequestError) -> Self {
+        omicron_common::api::external::Error::internal_error(&format!(
+            "{:#}",
```

Comment:
I don't think `external::Error` supports an alternate format; this could be just `{error}`. Arguably we should use slog-error-chain here to print the chain, but in practice it doesn't matter currently, because `external::Error` doesn't have any cases with chained errors.

Reply:
A quick search shows we have `:#` forms littered throughout the codebase, although arguably this doesn't matter as it just ends up being a no-op.

Reply:
Well, we also have `anyhow::Error`s littered throughout the codebase too. For those it _does_ matter. :man_shrugging: